cli/src/commands/invite/claim.rs

```diff
@@ -153,6 +245,23 @@ async fn step2_device(ctx: DeviceClaimInProgress1Ctx) -> anyhow::Result<DeviceCl
     Ok(ctx.do_signify_trust().await?)
 }
 
+/// Step 2: signify trust
+async fn step2_shamir(
+    ctx: ShamirRecoveryClaimInProgress1Ctx,
+) -> anyhow::Result<ShamirRecoveryClaimInProgress2Ctx> {
+    let sas_codes = ctx.generate_greeter_sas_choices(3);
+
+    let selected_sas = Select::new()
+        .items(&sas_codes)
+        .with_prompt("Select code provided by greeter")
+        .interact()?;
```

Comment:
Update the previous selectors for consistency, I recommend creating an helper function

Reply:
see #9334